lib/travis/build/git/clone.rb

```diff
@@ -56,6 +72,14 @@ def quiet?
             config[:git][:quiet]
           end
 
+          def sparseCheckout
+            config[:git][:sparseCheckout]
+          end
+
+          def sparseCheckout?
+            !!config[:git][:sparseCheckout]
```

Comment:
Sorry about the confusion. I'm just suggesting a replacement of the line here. If `sparseCheckout` is an empty string, this definition will return `true`.

In other words (note my previous error in code):

```ruby
def sparseCheckout?
  ! config[:git][:sparseCheckout].to_s.empty?
end
```